src/component/bom.js

```diff
@@ -2519,6 +2588,9 @@ const RescoringCell = ({
 
   const title = findingTypeToDisplayName(type)
 
+  const osInfo = osData?.data.os_id
```

Comment:
so we have two different cases of absent os-id (no data at all vs. an object where all attribute was explicitly set to null)? in that case, i think it would be nice to express this (e.g. using two booleans: `have_os_id`, `have_empty_os_id`, or by using an "enum"). I think it will make the code a lot easier to understand if we are more explicit about this (a short docstr might also help)